9/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM ubuntu:16.04
 
-ADD config/redis.config /etc/redis/redis.config
-ADD start /start
+COPY config/redis.config /etc/redis/redis.config
```

Comment:
I guess its just a tradeoff... we had to rework the dockerfile to make it tolerable to develop with. Happy to change it to whatever you prefer though